src/Features/VisualBasic/Portable/VBFeaturesResources.resx

```diff
@@ -1239,6 +1239,35 @@ Sub(&lt;parameterList&gt;) &lt;statement&gt;</value>
   <data name="Make_0_inheritable" xml:space="preserve">
     <value>Make '{0}' inheritable</value>
   </data>
+  <data name="Apply_file_header_preferences" xml:space="preserve">
+    <value>Apply file header preferences</value>
+  </data>
+  <data name="Apply_object_collection_initialization_preferences" xml:space="preserve">
+    <value>Apply object collection initialization preferences</value>
+  </data>
+  <data name="Apply_this_qualification_preferences" xml:space="preserve">
+    <value>Apply Me qualification preferences</value>
```

Comment:
value doesn't match data.  data says 'this' but value says 'me'.

Reply:
The data name matches what is in editor.Config which is common with C#, the Value is what should appear in the Option configuration for VB Code Cleanup.

Reply:
> The data name matches what is in editor.Config

these are not editorconfig key names.  these are resource string names.  resource string names should match the resource string value.

Reply:
Agree, for this resources file the pattern for new strings is to match the value

Reply:
OK, how do they map to Editor.Config is that done elsewhere?

Reply:
Fixed